tiling_segmentation.py

```diff
@@ -1,5 +1,6 @@
 from datetime import datetime
 import multiprocessing
+from numbers import Number
```

Comment:
Why import NUmber?

Reply:
Used in type hints. This class is more permissive. Suggests that input should be either a float or int without have to explicitly allow these two types.